beets/mediafile.py

```diff
@@ -77,6 +78,7 @@
     'mpc':  'Musepack',
     'asf':  'Windows Media',
     'aiff': 'AIFF',
+    'dsf':  'DSD Stream File',
```

Comment:
Arg, you're right—it's weird that there are competing standards here. But I agree in that case; let's keep the full name.